simulator-core/build.gradle

```diff
@@ -48,4 +59,60 @@ dependencies {
 
 test {
     useJUnitPlatform()
+}
+
+jar {
```

Comment:
do I correctly understand that it will produce a fat jar?

Reply:
correct.

Reply:
I'd rather release it as jar with dependencies. It will be lighter and give people some flexibility in dependency management. Although not sure what's the common pattern for releases.

Reply:
<img width="1201" alt="Screenshot 2019-04-16 at 14 24 11" src="https://user-images.githubusercontent.com/47893389/56209719-7b14f080-6054-11e9-82e4-f02a3fcc8314.png">

We release three things:
- runnable fat jar (7.8mb on the picture)
- java sources
- javadocs

instead of runnable fat jar, we could just release a jar without any dependancies but then people won't be able to run it / you will get ClassNotFoundException when trying to start it.

Reply:
Edit: nevermind I misunderstood your point, we have a pom file with depenendancies in our repo, so no need for a fat jar it only slows down things.